### PR TITLE
#160747011 Refactor code for improving functionality.

### PR DESCRIPTION
**Pull request by billkabanga**
### What does this PR do?
Refactor code to improve the responses from the server.
#### Description of Task to be completed?
Change the `page_not_found` method to `wrong_url`
Change the message from `Page not found` to `Wrong URL entry`
Change Base URL for tests to ` /api/v1/orders`
Modify the long lines of code in `views.py`
Remove the trailing new lines and white spaces.
#### How should this be manually tested?
Following the ReadMe for the API-feature branch, this can be tested through postman
#### What are the relevant pivotal tracker stories?
#160747011
#### Screenshot

pmn1

### CODE REVIEW

**Comment by Sebuliba-Adrian** [comment]
LGTM 👍

**Review by Sebuliba-Adrian — APPROVED** [review]
LGTM

File: api/views.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def post(self):
         method to post a question
         """
         args = self.reqparse.parse_args()
-        if not re.match(r"^[a-zA-Z]+$", args['client']):
+        if not re.match(r"^[a-zA-Z ]+$", args['client']):
             return make_response(jsonify({'message': 'Username should only have letters'}), 400)
```
Comment (kdaglas) [inline api/views.py:39]: this is really good and precise... the regexp are precise which is really good

**Comment by kdaglas** [comment]
I like your pull request format... good work

**Pull request merged**